cmd/confluent/main.go

```diff
@@ -14,19 +14,24 @@ import (
 
 // Injected from linker flags like `go build -ldflags "-X main.version=$VERSION" -X ...`
 var (
-	version = "v0.0.0"
-	commit  = ""
-	date    = ""
-	isTest  = "false"
+	version        = "v0.0.0"
+	commit         = ""
+	date           = ""
+	disableUpdates = "false"
+	isTest         = "false"
 )
 
 func main() {
 	cfg, err := load.LoadAndMigrate(v1.New())
 	cobra.CheckErr(err)
 
+	disableUpdates, err := strconv.ParseBool(disableUpdates)
```

Comment:
Just to double check my reading of these changes: users can't configure disable updates in their configs; updating is always enabled for archive/binary downloads and always disabled for package manager installs to not interfere with the package manager update feature.

Reply:
Yeah, on second thought, we should remove this from the configuration file altogether (instead of omitempty, use `json:"-"`)

Reply:
If a Homebrew user set `"updates_disabled": false`, it would reset to true the next time they ran any command, which could be disorienting.

Reply:
Never mind, that would be a breaking change.